### PR TITLE
Additional Remarks - Changes in behavior - Social 781a 

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Option to enter additional remarks in behavior - particularly social behavior changes.
These questions are asked after Additional Event screen, physical, mental, and work behavior changes screens.
```bah-781a-additionalRemarks-social```
http://localhost:3001/disability-benefits/apply/form-526-all-claims/new-disabilities/ptsd-781a-social-changes
## Testing done
tests/pages/socialBehaviorChanges.unit.spec.jsx

## Screenshots
screen shot 2018-11-16 at 8 17 41 am

## Acceptance criteria
- [ ] The PTSD type classification is correctly populated on page header based on the user's selection at the beginning of the workflow
- [ ] User is able to indicate changes to social habits as a result of the stressful incident per the Content Document
**NOTED: No error messages will appear because this form has no required fields.**

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by igreulich — APPROVED** [review]

**Comment by imeecuison** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by rianfowler — APPROVED** [review]

**Review by cvalarida — APPROVED** [review]
Looks good! I think it could use one more test to be safe, but that should be pretty straightforward.

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 1]
```diff
@@ -301,6 +302,13 @@ const formConfig = {
           uiSchema: physicalHealthChanges.uiSchema,
           schema: physicalHealthChanges.schema,
         },
+        socialBehaviorChanges: {
+          title: 'Additional Remarks - Social Behavior Changes',
+          path: 'new-disabilities/ptsd-781a-social-changes',
+          depends: isAnswering781aQuestions(0),
```
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/config/form.js]: Yes. @cvalarida : that's right.

File: src/applications/disability-benefits/all-claims/tests/pages/socialBehaviorChanges.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { DefinitionTester } from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import { mount } from 'enzyme';
+import formConfig from '../../config/form';
+
+describe('Work Behavior Changes 781a', () => {
```
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/tests/pages/socialBehaviorChanges.unit.spec.jsx]: Changed. Thank you!

File: src/applications/disability-benefits/all-claims/tests/pages/socialBehaviorChanges.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        definitions={formConfig}
+        schema={schema}
+        uiSchema={uiSchema}
+        data={{
+          socialBehaviorChanges: {
+            other: true,
```
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/tests/pages/socialBehaviorChanges.unit.spec.jsx]: Test addressing this has been added. Thank you!

**Pull request merged**